x/claims/genesis.go

```diff
@@ -40,7 +40,12 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, data types.GenesisState) {
 		}
 		k.SetClaimsRecord(ctx, addr, cr)
 
-		sumClaims = sumClaims.Add(cr.InitialClaimableAmount)
+		numActions := sdk.NewInt(int64(len(claimsRecord.ActionsCompleted)))
+		for _, claimed := range claimsRecord.ActionsCompleted {
+			if !claimed {
+				sumClaims = sumClaims.Add(cr.InitialClaimableAmount.Quo(numActions))
+			}
```

Comment:
I think we need to consider decay in here as well.. more complex than I thought:

https://github.com/tharsis/evmos/blob/c967e149f03204dc274284f314372c4e6591d21b/x/claims/keeper/claim.go#L11

Reply:
Solution is to just call this function with its inputs. Then we don't have to do all these weird calculations to try and do this correctly.

Reply:
Most functions preferred to be called with their inputs .. *heyooooo*

Reply:
@khoslaventures, decay is not considered in the escrowed account, the original solution is correct